oauth/tool/src/java/org/sakaiproject/oauth/servlet/AuthorisationServlet.java

```diff
@@ -171,6 +173,7 @@ private void sendToAuthorisePage(HttpServletRequest request, HttpServletResponse
 
             request.getRequestDispatcher(authorisePath).forward(request, response);
         } catch (OAuthException e) {
+            log.warn("Problem with oauth request_token", e);
```

Comment:
You could probably just log the message without having the whole stack here. Not sure the value of that in the logs. 

```suggestion
            log.warn("Problem with oauth request_token: {}", e.getMessage());
```